src/main/java/uk/gov/hmcts/reform/sscs/helpers/CaseFlagMappingServiceHelper.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.sscs.helpers;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@SuppressWarnings("PMD")
+public class CaseFlagMappingServiceHelper {
+
+    private final String[] mappingValues =
+        {
+        "disabledAccess",
+        "signLanguageType",
+        "hearingLoop",
+        "isConfidentialCase",
+        "dwpUCB",
+        "dwpPHME",
+        "urgentCase",
+        "adjournCaseInterpreterLanguage"
+        };
+
+    private final Map<String, Object> caseFlags;
```

Comment:
I think we can make this a Map<String, String> type

Reply:
Commented below.